Dockerfile

```diff
@@ -27,6 +27,7 @@ RUN mkdir /opt/ccache && ccache --set-config=cache_dir=/opt/ccache
 ENV PATH /opt/conda/bin:$PATH
 
 FROM dev-base as conda
+ARG PYTHON_VERSION=3.8
```

Comment:
nit: not an expert in Dockerfiles. but wouldn't `ARG PYTHON_VERSION` be enough here? (since `ARG PYTHON_VERSION=3.8` already defined at the top as default? makes it easier to manage later IMO